cms/envs/common.py

```diff
@@ -266,6 +266,14 @@
 CSRF_COOKIE_AGE = 60 * 60 * 24 * 7 * 52
 
 
+######################### Django Rest Framework ########################
```

Comment:
This looks identical to the LMS definition. I believe that the convention in such cases is to import the LMS setting rather than redefining it:

https://github.com/edx/edx-platform/blob/master/cms/envs/common.py#L36